app/views/videos/_video.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1 class="p-name"><%= link_to_if linked, video.title, video_path(video.slug) %></h1>
+
```

Comment:
And can you add a crumb trail nav to `videos`, like the book pages have to `books`?

<img width="859" alt="screen shot 2017-01-17 at 8 06 48 am" src="https://cloud.githubusercontent.com/assets/4361/22028868/88fb0696-dc8d-11e6-8103-0f2d64a58156.png">